docs/docs/Components/components-vector-stores.md

```diff
@@ -78,6 +80,54 @@ For an example of using the **Astra DB Vector Store** component with an embeddin
+The default model is `nvidia/llama-3.2-nv.reranker`.
+
+**Hybrid search** passes a vector similarity search and a lexical search to the [find and rerank Data API endpoint](https://docs.datastax.com/en/astra-db-serverless/api-reference/document-methods/find-and-rerank.html), where a reranker combines and sorts the results based on the vector search's similarity to the lexical terms.
+
+To use **Hybrid search** in the **Astra DB** component, do the following:
+
+1. To create a **Hybrid Search** template flow, click **New Flow** > **RAG** > **Hybrid Search**.
+2. In the **OpenAI** model component, add your **OpenAI API key**.
+3. In the **Astra DB** vector store component, add your **Astra DB Application Token**.
+4. In the **Database** field, select your database.
+5. In the **Collection** field, select the collection you want to search.
+6. In the **Playground**, enter a question about your data, such as `What are the features of my data?`
+Your query is sent to two components: an **OpenAI** model component, and the **Astra DB** vector database component.
+The **OpenAI** component contains a prompt for creating the lexical query from your input:
+```text
+You are a database query planner that takes a user requests, and converts to a search against the subject matter in question.
+You should convert the query into:
+1. A list of keywords to use against a Lucene text analyzer index, no more than 4. Strictly unigrams.
+2. A question to use as the basis for a QA embedding engine.
+Avoid common keywords associated with the user's subject matter.
+```
+7. To view the keywords and questions the **OpenAI** component generates from your collection, in the **OpenAI** component, click <Icon name="TextSearch" aria-label="Inspect icon" />.
+```
+1. Keywords: features, data, attributes, characteristics
+2. Question: What characteristics can be identified in my data?
+```
+8. To view the [DataFrame](/concepts-objects#dataframe-object) generated from the **OpenAI** component's response, in the **Structured Output** component, click <Icon name="TextSearch" aria-label="Inspect icon" />.
+The DataFrame is passed to a **Parser** component, which parses the the contents of the **Keywords** column into a string.
+
+This string of comma-separated words is passed to the **Lexical Terms** port of the **Astra DB** component.
+
+Note that the **Search Query** port of the Astra DB port is connected to the **Chat Input** component from step 6.
+This **Search Query** is vectorized, and both the **Search Query** and **Lexical Terms** content are sent to the reranker at the `find_and_rerank` endpoint.
+
+9. The reranker compares the vector search results against the list of terms from the lexical search.
+The highest-ranked results of your hybrid search are returned to the **Playground**.
+
+For more information, see the [DataStax documentation](https://docs.datastax.com/en/astra-db-serverless/api-reference/document-methods/find-and-rerank.html).
```

Comment:
The steps above talk more about the Open AI component than hybrid search/find-and-rerank command. I am not sure this link will be munch help if they have questions about the component. 

If you want to direct them to information about what happens during a hybrid search and how the reranking is calculated, link to the hybrid-search.html page I referenced in a previous comment.

Reply:
This is from me sharing the link with Mendon when we weren't sure about the timing of the general conceptual article. Agree we can update to:
```suggestion
For more information, see the [DataStax documentation](https://docs.datastax.com/en/astra-db-serverless/databases/hybrid-search.html).
```